gateway-ha/src/main/java/io/trino/gateway/ha/router/RoutingManager.java

```diff
@@ -125,6 +154,17 @@ public String findBackendForQueryId(String queryId)
         return backendAddress;
     }
 
+    public String findBackendForCookie(String cookie)
```

Comment:
```suggestion
    public String findBackend(String cookie)
```

Reply:
We also have a `findBackendForQueryId` in `RoutingManager` that takes a string, so we need to disambiguate them